### DISCUSSION TITLE
q-uploader sign shows after loading max-files from vuex

### DISCUSSION THREAD

**Discussion by tarikrital** [quasarframework/quasar#14402] [General - Components / Directives / etc] [answered]
First thank you Quasar team for the collection of components.

I have a page with 4 steps and dynamic q-uploader with max-files it works fine and hides the plus sign after having i.e 3 items added when going back (step 3) and again to 4 the items loaded from Vuex but the ➕ shows ! is it something missing or a bug?

**Comment by pdanpdan** [comment]
We need a reproduction codepen

**Comment by tarikrital** [comment, marked as answer]
I have it working by passing `scope` to delete func and using `scope.removeFile(file)`.
